templates/admin/user/edit.tmpl

```diff
@@ -196,14 +196,34 @@
 	</div>
 </div>
 
-<div class="ui small basic delete modal">
+<div class="ui small basic delete modal" id="delete-user-modal">
 	<div class="ui icon header">
 		{{svg "octicon-trash"}}
 		{{.i18n.Tr "settings.delete_account_title"}}
 	</div>
 	<div class="content">
 		<p>{{.i18n.Tr "settings.delete_account_desc"}}</p>
 	</div>
-	{{template "base/delete_modal_actions" .}}
+	<form class="ui form" method="POST" action="{{.Link}}/delete">
+		{{$.CsrfTokenHtml}}
+		<input type="hidden" name="id">
+		<div class="field">
+			<div class="ui checkbox">
+				<label for="purge">{{.i18n.Tr "admin.users.purge"}}</label>
+				<input name="purge" type="checkbox">
+			</div>
+			<p class="help">{{.i18n.Tr "admin.users.purge_help"}}</p>
+		</div>
+		<div class="text right actions">
+			<div class="ui red basic inverted cancel button">
+				{{svg "octicon-x"}}
+				{{.i18n.Tr "modal.no"}}
+			</div>
+			<button class="ui green basic inverted ok button">
+				{{svg "octicon-check"}}
+				{{.i18n.Tr "modal.yes"}}
+			</button>
+		</div>
```

Comment:
Is there a reason not to call `templates/base/delete_modal_actions.tmpl` instead?

Reply:
This PR predates that by quite some time.

Reply:
Makes sense...